pkg/sources/s3/s3.go

```diff
@@ -131,6 +131,7 @@ func (s *Source) setMaxObjectSize(maxObjectSize int64) {
 func (s *Source) newClient(region, roleArn string) (*s3.S3, error) {
 	cfg := aws.NewConfig()
 	cfg.CredentialsChainVerboseErrors = aws.Bool(true)
+	cfg.LogLevel = aws.LogLevel(aws.LogDebugWithRequestErrors)
```

Comment:
Was this just for testing or are we keeping it in?

Reply:
I thought it would be useful even for non-debugging purposes since it would only log during errors. What do you think?

Reply:
Yeah, makes sense to me.